backend/api-design.md

```diff
@@ -180,9 +197,17 @@ Creates a tournament match. With opponents, ... :TODO
 Returned HTTP codes:
 - 200: Match created successfully
 - 404: Tournament with that ID could not be found
-- 403: User not authorized
+- 401: User not authorized
 - 400: Missing required attributes in the provided payload
 
+#### GET `/tournaments/<id>/matches`
+
+Gets all tournament matches.
+
+Returned HTTP codes:
+- 200: Request registration successfully
```

Comment:
The endpoint talks about matches but the text talks about registration -> copy paste error.